geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -170,6 +171,11 @@ Other configuration
 You also have to configure the `vars/srid`, `vars/alternate_projections`, `vars/resolutions` and
 `vars/extent` they will be dispatched using `c2ctemplate` or YAML link.
 
+Now we can esily switch from window / grid display query result by using `gmfQueryGrid` configuration
+variable.
+
+Now we can eayly switch from flush non flush theme mode by using `gmfTreeManagerModeFlush` configuration variable,
```

Comment:
"Now we can eayly switch from flush non flush theme mode" -> "Now you can easily switch between flush theme mode and non-flush theme mode"